hdw.dat.rkn

```diff
@@ -8,6 +8,10 @@
 #
 #  1   2      3        4         5       6      7      8    9   10   11   12  13(1)  13(2)  13(3)  14   15  16 17
 #
+# UNTIL 11/Sep/2021 20:00:00.0 Date that Borealis installed at rkn
+   65 2021 21960000 62.828  -92.113    50.0    5.7  3.24  1   10  0.000  1    0.0 -100.0    0.0 0.0  0 225 16
```

Comment:
I think this `yrsec` corresponds to `20210912 04:00:00 UT`, rather than `20210911 20:00:00 UT` as the comment line indicates.  Should the `yrsec` instead be `21931200`?